Formula/c/coder.rb

```diff
@@ -21,13 +21,17 @@ def install
     ldflags = %W[
       -s -w
       -X github.com/coder/coder/v2/buildinfo.tag=#{version}
+      -X github.com/coder/coder/v2/buildinfo.agpl=true
```

Comment:
What does this actually do?

Reply:
`coder version` outputs some licensing information, and it uses this variable to determine which license is in use.

Reply:
https://github.com/coder/coder/blob/main/scripts/build_go.sh#L110